herbert_core/applications/docify/private/get_builtin_handle.m

```diff
@@ -0,0 +1,15 @@
+function handle = get_builtin_handle(fn_name)
```

Comment:
Don't know... maybe? Where else would it be needed?

I'm also thinking of modifying it so that instead of finding the built-in `help` or `doc` it just returns the next function in the list of `which(func, '-all')` after the current function - this would enable nested `help` / `doc` commands (the original idea for this is for the Python wrapper functions for euphonic and brille, in those cases the `help` / `doc` commands calls the Python `pydoc` functions to get the help instead).

Reply:
Just generally my philosophy is that if something is non-conflicting (i.e overriding/specific use case) and may be used elsewhere reasonably it should be accessible to avoid any excuses of having 300 instances of the same code.